backend/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# docs:  http://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+indent_style = tab
+indent_size = 4
+
+[*.{js,ts}]
+quote_type = double
+continuation_indent_size = 4
+curly_bracket_next_line = false
+indent_brace_style = BSD
```

Comment:
```suggestion
```

Reply:
Please install editorconfig extension as well so that these can be taken care of automatically.

Eg: New line is missing in this file itself and could be taken care by the extension.